tests/foreman/ui/test_reporttemplates.py

```diff
@@ -233,6 +234,31 @@ def test_positive_generate_registered_hosts_report(session, module_org, module_l
             assert res['Operating System'] == '{0} {1}'.format(os_name, os.major)
 
 
+@upgrade
+@tier2
+def test_positive_generate_subscriptions_report_json(session, module_org, module_loc):
+    """Use provided Subscriptions report, generate JSON
+
+    :id: b44d4cd8-a88e-47cf-9993-0bb871ac2c96
+
+    :expectedresults: The Subscriptions report is generated in JSON
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    # generate Subscriptions report
+    with session:
+        file_path = session.reporttemplate.generate("Subscriptions",
+                                                    values={'output_format': 'JSON'})
```

Comment:
Shouldn't you first upload manifest, or create product?
Looks like this report is using subscriptions from all organizations, but if there is no manifest in your Satellite at all, generated file will just contain `[]` (empty array).

Reply:
I thought there is some manifest already in the environment when the tests are run. Isn't that the case?

Reply:
I don't think we upload manifest as part of environment preparation.

If manifest will be already available when this test is run, it's most likely only because some other test uploaded it. Depending on test execution order has many drawbacks, which I probably don't have to state here.

If you are confident this test will always run after some other test uploading manifest, and you want to leave it as that, I won't block you.